care/facility/models/mixins/permissions/facility.py

```diff
@@ -7,6 +7,21 @@ class FacilityPermissionMixin(BasePermissionMixin):
     def has_bulk_upsert_permission(request):
         return request.user.is_superuser
 
+    @staticmethod
+    def has_write_permission(request):
+        from care.users.models import State, District
+        state = State.objects.get(id=request.data["state"])
+        district = District.objects.get(id=request.data["district"]) 
+        if (
+            request.user.is_superuser or (
+                request.user.user_type <= User.TYPE_VALUE_MAP["DistrictAdmin"]
```

Comment:
There can be dist admins trying to create facilities in other districts and state admins trying to create in another state, we need to add those to the conditions as well.

Reply:
@vigneshhari Should I restrict localbodyAdmin and wardAdmin too? Because they are also able to create facilities in other localbody or ward.